packages/commonwealth/server/routes/status.ts

```diff
@@ -134,6 +136,14 @@ const getChainStatus = async (models: DB) => {
   };
 };
 
+export const getChainActivity = new Activity(
+  'getChainStatus',
+  getChainStatus,
+  'getChainStatus',
+  60 * 5, // 5 minutes
```

Comment:
we can keep a higher TTL if we like, else its fine too, as raw query performance is improved too. Thanks!